### PR TITLE
Better presentation for suggestions & fixing indexing bug on suggestions

### LINKED ISSUE #73: Make fixed search bar size when using suggestions
When using suggestions, the relative size of the search bar component makes for a bad user experience. The value should be fixed.

### PR DESCRIPTION
**Pull request by Samuelfaure**
1/ Fixed suggestion making search component change size too much (fix #73 )
2/ Fixed bug on suggestion index not being computed correctly
3/ Now resetting index of suggestions when changing fullText
4/ Preventing keydown.up default behavior putting cursor back on searchbar input

fixedsize

### CODE REVIEW

File: src/components/search/SearchBar.vue [hunk 1]
```diff
@@ -16,8 +16,10 @@
           v-bind:class="{'active': suggestActive(index)}">
         <span>{{ suggestion | capitalize | removeExtraChars}}</span>
       </li>
+      <!-- Filling with hidden divs so search bar will always be same size -->
+      <!-- eslint-disable-next-line -->
+      <li class="suggestion__box_hidden" v-for="index in suggestionNumberToMax"></li>
```
Comment (thimy) [inline src/components/search/SearchBar.vue]: Would be better to add the `suggestion__box` class and add a generic `hidden` class that doesn't extend the style, as it duplicates code.

**Comment by Samuelfaure** [comment]
Added demo GIF

**Comment by thimy** [comment]
Could you make the suggestions take the whole input width?

**Comment by Samuelfaure** [comment]
Yes!

**Pull request merged**